### ISSUE TITLE
Funds are blocked on Copay - Multisig wallet

### ISSUE THREAD

**Issue by PotterGGG** [bitpay/copay#9074]
My money on copey is blocked

**Comment by BGalliotti** [comment]
Hello @PotterGGG 
Thanks for reaching out to us.

Are you getting any errors?

**So we can further help, please provide as many of the following details as you can:**

- Current device used: Phone model 
- Current operating system version for the device: iOS, Android..
- Current version of the wallet used: CoPay App > Settings > About CoPay 
- Any log files containing error details: CoPay App > Settings > About CoPay > Session Log > Tap on the Upper right corner (three dots) > Send by email (you can screenshot the content as well and attach the images to your e-mail. In such case, please make sure you move the slider to the **"Debug"** position).
- Error screenshot: On [iPhone](https://apple.co/1DAB204) on [Android](https://www.wikihow.tech/Take-Screenshots-on-an-Android)
- Steps to replicate: Steps needed for the error to appear.
- CoPay Wallet ID - To get your Wallet ID, open your CoPay application, then go to `Settings` > [Your wallet] > `More Options` > `Wallet Information`. On your phone, if you tap on the wallet ID it will copy it to the clipboard so you can paste in your e-mail.

**We look forward to hearing from you.**

Please let us know if you have any additional questions.
Kind regards.

**Comment by PotterGGG** [comment]
Iphone 6
iOS
Current version : v4.6.1
Log files sent in prvious email
It is written under my funds: All funds are blocked
ID   19143e47-c993-4788-8e30-9bcef2eb256a

чт, 30 авг. 2018 г. в 17:51, BGalliotti <notifications@github.com>:

**Comment by BGalliotti** [comment]
Hello @PotterGGG 
Thanks for reaching out to us.

This is your balance:

* Balance
  Total: 0.02113400 btc (0.02113400 btc locked)
  Confirmed: 0.02113400 btc (0.02113400 btc locked)
  Available: 0.00000000 btc (0.00000000 btc confirmed / 0.00000000 btc unconfirmed)

And this is why your funds are locked:

* Pending tx proposals
  [Tue Aug 14 2018 12:06:32 GMT-0400] 0.02000000 btc (fee/kb 0.00039654 btc) => 1A5xNfFfHoZF4c7mr57u9szy5JyVBhXU6m (status: missing 1 signatures)

Basically you are trying to send funds, and the funds are locked since the proposal is still waiting for one signature.

If you cancel the proposal,you wil be able to use the funds, but remember you need two signatures for the proposal to be approved and the funds sent.

Please let us know if you have any additional questions.
Kind regards.
